sample-app/src/components/SearchBar.tsx

```diff
@@ -6,28 +6,38 @@ import '../sass/SearchBar.scss';
 import '../sass/Autocomplete.scss';
 import LoadingIndicator from './LoadingIndicator';
 import { useAutocomplete } from '../hooks/useAutocomplete';
+import { useRef } from 'react';
+import { AutocompleteResponse } from '@yext/answers-headless';
+import { executeSearchWithIntents } from '../utils/search-operations';
 
 const SCREENREADER_INSTRUCTIONS = 'When autocomplete results are available, use up and down arrows to review and enter to select.'
 
 interface Props {
   placeholder?: string,
   isVertical: boolean,
+  geolocationOptions?: PositionOptions,
```

Comment:
do we need to add this to the props, or can we use the default for now?

Reply:
Tom suggested to add this since the SDK's SearchBar allows you to configure the GeoLocation options